localstack/aws/forwarder.py

```diff
@@ -160,12 +160,11 @@ def create_aws_request_context(
     # we re-use botocore internals here to serialize the HTTP request,
     # but deactivate validation (validation errors should be handled by the backend)
     # and don't send it yet
-    client = aws_stack.connect_to_service(
-        service_name,
+    client = connect_to(
         endpoint_url=endpoint_url,
         region_name=region,
         config=_non_validating_boto_config,
-    )
+    ).__getattr__(service_name)
```

Comment:
Not a fan of this, for this case we have `connect_to.get_client(`

Reply:
Changed in aa10884a80270f1bcee30bf94ff8f8341313f3cc :+1: